src/application/Program.java

```diff
@@ -21,7 +21,7 @@ public static void main(String[] args) {
 		List<ChessPiece> captured = new ArrayList<>();
 
 
-		while (true){
+		while (!chessMatch.getCheckMate()){
```

Comment:
é uma boa prática e facilita a leitura "Evitar sentenças negativas"

https://www.obsidiansecurity.com/blog/why-to-stop-writing-negative-code/